### DISCUSSION TITLE
How to query document by document ids?

### DISCUSSION THREAD

**Discussion by OHeroJ** [appwrite/appwrite#7296] [Q&A]
### 🔖 Feature description

how to query document by document ids 

```js
const finalIds = ['xxxxxxxxxxxxxxxxxxxxxxx', 'xxxxxxxxxxxxxxxxxxxxbbbbb', ] // document ids
let data = await appwrite_databases.listDocuments('Databse ID', 
		'Collection ID', [
		Query.equal(r'$collectionId', finalIds)
]);
```

but reuturn 400 status

### 🎤 Pitch

return documents

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
Since you're trying to filter by `$id`, can you try using `$id` in the query? Something like:

```javascript
const finalIds = ['xxxxxxxxxxxxxxxxxxxxxxx', 'xxxxxxxxxxxxxxxxxxxxbbbbb', ] // document ids
let data = await appwrite_databases.listDocuments(
  'Database ID',
  'Collection ID', 
  [
    Query.equal(r'$id', finalIds)
  ]
);
```
